Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -531,17 +531,14 @@ void SoftwareGLContext::gl_enable(GLenum capability)
         m_cull_faces = true;
         break;
     case GL_DEPTH_TEST:
-        m_depth_test_enabled = true;
```

Comment:
I'd much rather keep all this state local to LibGL. This is part of the OpenGL state machine and we don't want to bother the driver all the time once we talk to real hardware.

Reply:
There's a lot of information in `RasterizerOptions` already that could be in the state machine. In the commit message, I was alluding to those options already being part of the state machine and that they could/should be decoupled from `SoftwareRasterizer`. With that in mind, what do you think?

Reply:
Some context from a discussion on Discord: @sunverwerth is working on a new workflow where the context's state would be compiled into a rasterizer state just before rasterization. In this case, the state would be present in both the context and the rasterization driver. Commit is removed :-)